base/applications/network/dwnl/lang/ro-RO.rc

```diff
@@ -0,0 +1,32 @@
+/*
+ * PROJECT:     ReactOS File Download Utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Romanian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Uz: dwnl URL [DESTINAȚIE]\n"
+    IDS_RESOLVING "Rezolvare %s... "
+    IDS_DONE "terminat.\n"
+    IDS_LENGTH_FULL "Lungime: %I64u [%s]\n"
+    IDS_LENGTH "Lungime: %I64u\n"
+    IDS_FILE_SAVED "\nFișier salvat.\n"
+    IDS_CONNECTING_TO_FULL "Conectare la %s [%s]... "
+    IDS_CONNECTING_TO "Conectare la %s... "
+    IDS_REDIRECTING_TO "Redirecționare la %s... "
+    IDS_SEND_REQUEST "Trimitere cerere... "
+    IDS_BYTES_DOWNLOADED "%I64u octeți descărcate"
+    IDS_BYTES_DOWNLOADED_FULL "%2d%% (%I64u octeți descărcate)"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_ERROR_DOWNLOAD "\nEROARE: Descărcare eșuată.\n"
+    IDS_ERROR_CODE "\nEROARE: %u\n"
+    IDS_ERROR_FILENAME "\nEROARE: Determinarea numelui de fișier este imposibilă, specificați destinația numelui de fișier.\n"
+    IDS_ERROR_PROTOCOL "\nEROARE: Protocolul nu este suportat.\n"
```

Comment:
Romanian "nerecunoscut" (considering we're talking about a protocol) may be a better term here (for the sake of consistency with the rest of ReactOS translated content, if nothing else).